pkg/api/grpcServer.go

```diff
@@ -71,11 +94,8 @@ func NewGRPCServer(aspectDispatcher adapterManager.AspectDispatcher, tracer trac
 }
 
 // dispatcher does all the nitty-gritty details of handling the mixer's low-level API
-// protocol and dispatching to the right API handler.
-func (s *grpcServer) dispatcher(stream grpc.Stream, methodName string,
-	getState func() (request proto.Message, response proto.Message, requestAttrs *mixerpb.Attributes, responseAttrs *mixerpb.Attributes, result *rpc.Status),
-	worker func(ctx context.Context, requestBag *attribute.MutableBag, responseBag *attribute.MutableBag,
-		request proto.Message, response proto.Message)) error {
+// protocol and dispatching to the right API dispatchWrapperFn.
+func (s *grpcServer) dispatcher(stream grpc.Stream, methodName string, getState stateGetterFn, worker dispatchFn) error {
```

Comment:
1. nit. rename this to `dispatch` 
2. I think it would be clearer if we make a call to `s.preprocess()` here.
I do like the python style func decoration with s.preprocess() !